src/redismodule.h

```diff
@@ -14,6 +14,10 @@
 /* API versions. */
 #define REDISMODULE_APIVER_1 1
 
+/* Version of the RedisModuleTypeMethods structure. Once the RedisModuleTypeMethods 
+ * structure is changed, this version number needs to be changed synchronistically. */
+#define REDISMODULE_TYPE_METHOD_VERSION 3
```

Comment:
why did you move that away from the struct?

Reply:
Because `#define REDISMODULE_CORE 1` is in server.h, as originally designed, I can't access `REDISMODULE_TYPE_METHOD_VERSION` in module.c